index.html

```diff
@@ -152,14 +151,16 @@
         font-size: 20px;
         margin-top: 3.5rem;
       }
-      .info__image {
-        width: 45%;
+      .info__iframe {
+        width: 560px;
+        height: 315px;
```

Comment:
Need to look at better solution, fixed values in large screen size don't look like before.

Reply:
It's fixed